packaging/greenboot/microshift_set_unhealthy.sh

```diff
@@ -15,13 +18,27 @@ if [ ! -f /run/ostree-booted ]; then
     exit 0
 fi
 
+# Define var FORCE to force overwrite
+if [[ ! -v FORCE ]] && [[ -e "${FILE}" ]]; then
+    h=$(jq -r '.health' "${FILE}")
+    b=$(jq -r '.boot_id' "${FILE}")
+    d=$(jq -r '.deployment_id' "${FILE}")
+
+    expected_backup="${DIR}/${d}_${b}"
+    if [[ "${h}" == "healthy" ]] && [[ ! -e "${expected_backup}" ]]; then
+        echo "State 'healthy' is persisted, but the backup was not created."
+        echo "Not overwriting the health.json to retry backup on next boot"
```

Comment:
```suggestion
        echo "Not overwriting the health.json, will retry backup on next boot"
```

Does this change convey the same meaning? If so, it's a little more explicit.